src/components/message-box/MessageBox.tsx

```diff
@@ -1,24 +1,45 @@
 import React from 'react';
-import {StyleProp, TouchableOpacity, View, ViewStyle} from 'react-native';
+import {
+  Linking,
+  StyleProp,
+  TouchableOpacity,
+  View,
+  ViewStyle,
+} from 'react-native';
 import {Statuses, StyleSheet, useTheme} from '@atb/theme';
 import ThemeText from '@atb/components/text';
 import ThemeIcon from '@atb/components/theme-icon';
 import MessageBoxTexts from '@atb/translations/components/MessageBox';
 import {useTranslation} from '@atb/translations';
 import {Close} from '@atb/assets/svg/mono-icons/actions';
-import {screenReaderPause} from '@atb/components/accessible-text';
 import {messageTypeToIcon} from '@atb/utils/message-type-to-icon';
+import {TouchableOpacityOrView} from '@atb/components/touchable-opacity-or-view';
+import insets from '@atb/utils/insets';
+import {screenReaderPause} from '@atb/components/accessible-text';
+
+/**
+ * Configuration for how the onPress on the message box should work. The
+ * configuration is split into action (function) or url (string) so that we can
+ * decide correct accessibility role on the message box.
+ *
+ * Note that for the screen reader the value in the `text` field will be
+ * prefixed with "Activate to". So good labels are for example "Try again",
+ * "Read more at atb.no" and "Show details" as they all work well with the
+ * prefix.
+ */
+type OnPressConfig = {
+  text: string;
+} & ({action: () => void} | {url: string});
 
 export type MessageBoxProps = {
   type: Statuses;
   title?: string;
   message: string;
   noStatusIcon?: boolean;
   onDismiss?: () => void;
-  onPress?: () => void;
-  onPressText?: string;
   isMarkdown?: boolean;
   style?: StyleProp<ViewStyle>;
+  onPressConfig?: OnPressConfig;
```

Comment:
Feel free to suggest better naming

Reply:
Good suggestion! I feel this would be just as good, but still not the perfect solution that I was dreaming of 😅 Will not use time on changing anything now.